mappings/com/google/common/base/Suppliers$MemoizingSupplier.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS afej com/google/common/base/Suppliers$MemoizingSupplier
```

Comment:
```suggestion
CLASS afej com/google/common/base/Suppliers_MemoizingSupplier
```

Despite what I said on Discord, it seems like JADX can't handle this correctly yet. We'll need to use underscores instead of dollar signs for now